cromulent/model.py

```diff
@@ -618,15 +645,15 @@ def _toJSON(self, done, top=False):
 					self.maybe_warn(msg)
 
 		# Add back context at the top, if set
-		if top and self._factory.context_uri: 
+		if top == self and self._factory.context_uri: 
```

Comment:
Should this test for identity and not equality (in case equality has been overridden)?

Reply:
if top is self ?